server/repository/src/db_diesel/number_row.rs

```diff
@@ -248,68 +254,87 @@ mod number_row_mapping_test {
 
-        let number_row_type = NumberRowType::Program("EXAMPLE_TEST".to_string());
-        match number_row_type {
-            NumberRowType::InboundShipment => {
-                assert!(
-                    NumberRowType::try_from(NumberRowType::InboundShipment.to_string()).unwrap()
-                        == NumberRowType::InboundShipment
-                )
-            }
-            NumberRowType::OutboundShipment => {
-                assert!(
-                    NumberRowType::try_from(NumberRowType::OutboundShipment.to_string()).unwrap()
-                        == NumberRowType::OutboundShipment
-                )
-            }
-            NumberRowType::InventoryAddition => {
-                assert!(
-                    NumberRowType::try_from(NumberRowType::InventoryAddition.to_string()).unwrap()
-                        == NumberRowType::InventoryAddition
-                )
-            }
-            NumberRowType::InventoryReduction => {
-                assert!(
-                    NumberRowType::try_from(NumberRowType::InventoryReduction.to_string()).unwrap()
-                        == NumberRowType::InventoryReduction
-                )
-            }
-            NumberRowType::Prescription => {
-                assert!(
-                    NumberRowType::try_from(NumberRowType::Prescription.to_string()).unwrap()
-                        == NumberRowType::Prescription
-                )
-            }
-            NumberRowType::RequestRequisition => {
-                assert!(
-                    NumberRowType::try_from(NumberRowType::RequestRequisition.to_string()).unwrap()
-                        == NumberRowType::RequestRequisition
-                )
-            }
-            NumberRowType::ResponseRequisition => {
-                assert!(
-                    NumberRowType::try_from(NumberRowType::ResponseRequisition.to_string())
-                        .unwrap()
-                        == NumberRowType::ResponseRequisition
-                )
-            }
-            NumberRowType::Stocktake => {
-                assert!(
-                    NumberRowType::try_from(NumberRowType::Stocktake.to_string()).unwrap()
-                        == NumberRowType::Stocktake
-                )
-            }
-            NumberRowType::Program(s) => {
-                assert!(
-                    NumberRowType::try_from(NumberRowType::Program(s.clone()).to_string()).unwrap()
-                        == NumberRowType::Program(s)
-                )
-            }
-            NumberRowType::Repack => {
-                assert!(
-                    NumberRowType::try_from(NumberRowType::Repack.to_string()).unwrap()
-                        == NumberRowType::Repack
-                )
+        for number_row_type in vec![
+            NumberRowType::Program("EXAMPLE_TEST".to_string()),
+            NumberRowType::OutboundReturn,
+            NumberRowType::InboundReturn,
```

Comment:
@andreievg why did you add the for loop here?

Reply:
Just want to add something here, from discussion, for future reference:

Not very strong on this, but kind of wanted this test to also check explicitly that ` "INBOUND_RETURN" => Ok(NumberRowType::InboundReturn),` was actually added, not sure if it's justified, since for test to work `NumberRowType::InboundReturn` would also need to be added to the array above. vs previous change where just adding the variant here would mean a comment above would be read and the extras added to try_from